### ISSUE TITLE
viz: mapdeck

### ISSUE THREAD

**Issue by bananaonthemoon** [bananaonthemoon/wybory#9]
jest w ogóle możliwe użycie tego pakietu do wyborów? bo wszystko, co znajduję opiera się na csv ze współrzędnymi. serio nie wiem co mam zrobić :(  próbowałam https://symbolixau.github.io/mapdeck/reference/add_grid.html ale to, co próbowałam, nie działa.

**Comment by Nowosad** [comment]
Probowała Pani zmienić układ współrzędny danych na WGS84 (4326)?

**Comment by bananaonthemoon** [comment]
tak, zmieniłam. koniec końców udało się coś wyświetlić, ale to są zwykłe poligony, więc trochę słabo. w dodatku teraz utknęłam przy paletach. chciałam zrobić gridy/hex używając tego pakietu, ale to nie jest możliwe (obliczane są na podstawie danych punktowych).
